recipes/maracluster/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: 'maracluster'
+  version: '0.05.0'
+
+source:
+  git_url: https://github.com/OpenMS/THIRDPARTY
+  git_rev: 8aac20e
+  md5: unused
+
+build:
+  number: 0
+
+requirements:
+  run:
+    - {{ compiler('cxx') }}c
```

Comment:
there is a c at the end ... put this into the build section and see if this works....

Reply:
hmm.

```
10:28:03 BIOCONDA INFO (OUT) maracluster: /lib64/libc.so.6: version `GLIBC_2.17' not found (required by maracluster)
10:28:03 BIOCONDA INFO (OUT) maracluster: /lib64/libc.so.6: version `GLIBC_2.14' not found (required by maracluster)
```

Reply:
Can it be that this binary was compiled against a newer libc version than we are using?
Can you build this package from source?

Reply:
Yep, over here https://github.com/bioconda/bioconda-recipes/pull/19219 but without success (there I tried 1.x but for OpenMS we should try 0.5).

Reply:
Not sure what we can do, besides recompiling ... we need to compile against older libc, otherwise, it will not work on older HPC.